fbt_lib/src/lib.rs

```diff
@@ -1,20 +1,27 @@
-use std::path::PathBuf;
-
 pub mod types;
 
-pub fn test_all() -> anyhow::Result<bool> {
+use std::time::{Duration, Instant};
```

Comment:
Remove this. Dont use `use` statements as much as possible.

Reply:
will keep in mind.